.circleci/scripts/binary_upload.sh

```diff
@@ -37,22 +39,15 @@ do_backup() {
 conda_upload() {
   (
     set -x
-    #Loop until counter is 3
-    counter=1
-    while [[ $counter -le 3 ]]
-    do
-      ${ANACONDA} \
-      upload  \
-      ${PKG_DIR}/*.tar.bz2 \
-      -u "pytorch-${UPLOAD_CHANNEL}" \
-      --label main \
-      --no-progress \
-      --force
 
-      rc=$?
-      # In case of failure, do a retry upload
-      [[ $rc -ne 0 ]] && ((counter++)) || break
-    done
+    retry \
```

Comment:
is retry 3 times by default?

Reply:
I think 3 times should be good, we don't want more in case if there is a real outage, and we don't want much less because we can have timeouts again

Reply:
Oh, I missed that retry was a function definition, and it was designed to be 3 times. I was thinking retry was a utility command.